.github/workflows/slack-notification.yml

```diff
@@ -0,0 +1,80 @@
+name: Slack Notification
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows:
+      - Cli Release
+    branches: [main]
+    types:
+      - completed
+jobs:
+  wait-for-releases:
+    name: Wait for Docker, Homebrew, RPM and Platform executables Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Extract branch name
+        id: extract_branch
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+      - name: Checkout cli repo
+        uses: actions/checkout@v2
+      - name: Wait for Docker Release
+        run: source .github/scripts/trigger-and-wait.sh
+        env:
+          INPUT_OWNER: twilio
```

Comment:
[nit] Can we keep the common env variables under job instead of repeating for every steps. info [here](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idenv)